tests/ui/neg_multiply.stderr

```diff
@@ -1,16 +1,16 @@
-error: negation by multiplying with `-1`
+error: this `multiplication with -1` can be written more succinctly
   --> $DIR/neg_multiply.rs:27:5
    |
 LL |     x * -1;
-   |     ^^^^^^
+   |     ^^^^^^ help: consider using: `-x`
    |
    = note: `-D clippy::neg-multiply` implied by `-D warnings`
 
-error: negation by multiplying with `-1`
+error: this `multiplication with -1` can be written more succinctly
   --> $DIR/neg_multiply.rs:29:5
    |
 LL |     -1 * x;
```

Comment:
Could you also add some test cases where the second factor is more than a value. Here are some examples you can take:

```rs
100 + x * -1;
(100 + x) * -1;
-1 * 17;
0xcafe | 0xff00 * -1
```

Then this should be sufficiently tested :upside_down_face:

Reply:
Seems good 👍